src/test/java/unicash/ui/StyleSheetTest.java

```diff
@@ -0,0 +1,154 @@
+package unicash.ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static unicash.ui.StyleSheet.BLUE_OFFSET;
+import static unicash.ui.StyleSheet.BRIGHTNESS_OFFSET;
+import static unicash.ui.StyleSheet.BRIGHTNESS_THRESHOLD;
+import static unicash.ui.StyleSheet.COLOR_END_INDEX;
+import static unicash.ui.StyleSheet.COLOR_START_INDEX;
+import static unicash.ui.StyleSheet.DEFAULT_BACKGROUND_COLOR_HEX;
+import static unicash.ui.StyleSheet.FONT_STYLE_BOLD;
+import static unicash.ui.StyleSheet.GREEN_OFFSET;
+import static unicash.ui.StyleSheet.IS_YELLOW_SKEW;
+import static unicash.ui.StyleSheet.MAX_COLOUR_VALUE;
+import static unicash.ui.StyleSheet.RED_OFFSET;
+import static unicash.ui.StyleSheet.TEXT_FILL_RED;
+import static unicash.ui.StyleSheet.USE_DEFAULT_STYLE;
+import static unicash.ui.StyleSheet.absoluteColorAddition;
+import static unicash.ui.StyleSheet.adjustBrightness;
+import static unicash.ui.StyleSheet.getColorOutput;
+import static unicash.ui.StyleSheet.padHexString;
+
+import org.junit.jupiter.api.Test;
+
+public class StyleSheetTest {
+
+    @Test
+    public void testConstants() {
+        assertEquals("-fx-text-fill: red", TEXT_FILL_RED);
+        assertEquals("-fx-font-weight: bold", FONT_STYLE_BOLD);
+    }
```

Comment:
Not sure if we do need to test the constants. These tests are quite inflexible as they don't test every constant and if edited to do so, it'll be a pain to maintain if we change the constants since we basically need to duplicate the values